src/Banner.js

```diff
@@ -91,7 +91,7 @@ Banner.propTypes = {
   iconName: PropTypes.string,
   onClose: PropTypes.func,
   showIcon: PropTypes.bool,
-  text: PropTypes.string,
+  text: PropTypes.node,
```

Comment:
I could be missing something important, but what is the benefit / reason of this prop type change for `text` prop?

Reply:
when running the tests, this line specifically https://github.com/pricelinelabs/design-system/blob/master/src/__tests__/Banner.js#L40, if throws a PropType warning when using JSX instead of a raw string

Reply:
Since there is an explicit test case for it, I figured it must be a use case that we intend to support.